.gitattributes

```diff
@@ -0,0 +1,15 @@
+# This file is documented at https://git-scm.com/docs/gitattributes.
+#
+# Linguist-specific attributes are documented at
+# https://github.com/github/linguist.
+
+# Collapse generated code in code reviews.
+**/zz_generated.*.go linguist-generated=true
+apis/generated/** linguist-generated=true
+apis/**/generated.pb.go linguist-generated=true
+apis/**/generated.proto linguist-generated=true
+config/crd/experimental/** linguist-generated=true
+config/crd/standard/** linguist-generated=true
+pkg/client/** linguist-generated=true
```

Comment:
No this path now, is it in plan?

Reply:
Good point, I missed that. We should remove this path until we actually generate things there, @sachinsmc.